### ISSUE TITLE
Crash when using newSMFeedbackFragmentInstance() and ChildFragmentManager

### ISSUE THREAD

**Issue by nicklasl** [SurveyMonkey/surveymonkey-android-sdk#60]
I get a crash when adding the fragment constructed by `SurveyMonkey.newSMFeedbackFragmentInstance(MY_HASH)` from within a dialogfragment, thus using that fragments `childFragmentManager`.

I would rather handle (or chose not to handle) the internet connectivity from my own app and not have this in your SDK.

```
java.lang.RuntimeException: Error receiving broadcast Intent { act=android.net.conn.CONNECTIVITY_CHANGE flg=0x4000010 (has extras) } in com.surveymonkey.surveymonkeyandroidsdk.SMFeedbackFragment$ConnectivityMonitor@cc6b1ca
        at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$-android_app_LoadedApk$ReceiverDispatcher$Args_52497(LoadedApk.java:1323)
        at android.app.-$Lambda$aS31cHIhRx41653CMnd4gZqshIQ.$m$7(Unknown Source:4)
        at android.app.-$Lambda$aS31cHIhRx41653CMnd4gZqshIQ.run(Unknown Source:39)
        at android.os.Handler.handleCallback(Handler.java:790)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.support.v4.app.FragmentActivity android.support.v4.app.Fragment.getActivity()' on a null object reference
        at com.surveymonkey.surveymonkeyandroidsdk.SMFeedbackFragment$ConnectivityMonitor.isNetworkConnected(SMFeedbackFragment.java:316)
        at com.surveymonkey.surveymonkeyandroidsdk.SMFeedbackFragment$ConnectivityMonitor.onReceive(SMFeedbackFragment.java:306)
        at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$-android_app_LoadedApk$ReceiverDispatcher$Args_52497(LoadedApk.java:1313)
```

**Comment by ayeung** [comment]
Added a fix for the issue you're seeing in the next release 1.0.7.  We currently look at the internet connectivity to provide the user with a relevant offline message, so I did not make any changes to that.

**Comment by ayeung** [comment]
Just released 2.0.0.  Check it out when you get the chance.  I'll be closing this issue for now, feel free to reopen and/or comment if you are still having issues.

Note that I did run into a situation where the previous version of the library was cached on Android Studio, so I would also invalidate the cache to make sure you are grabbing the latest if you run into any issues.

**Comment by liviu-timar** [comment]
Hi @ayeung,

Still getting the error even on v2.0.0. I'm using it in the same way, in a fragment, so with getChildFragmentManager.

Thank you!
